### ISSUE TITLE
[BUG] Sent messages are grayed out

### ISSUE THREAD

**Issue by dhoeld** [RocketChat/Rocket.Chat.iOS#2663]
**Details**
- Your Rocket.Chat app version: 3.4.1 (250)
- Your Rocket.Chat server version: 0.74.3
- Device (or Simulator) you're running with: Iphone 8 Plus (iOS 12.2)

**How to reproduce the problem:**
- Send some arbitrary messages to another Rocket.Chat user

**Screenshot:**
![rchat_screenshot](https://user-images.githubusercontent.com/6930035/56048624-303f6400-5d48-11e9-86a6-4e40a585c4d7.PNG)

**Comment by rafaelks** [comment]
@dhoeld Usually the grayed messages aren't sent yet, they are temporary messages... do you see this messages in other clients?

**Comment by dhoeld** [comment]
@rafaelks Yes, you can see this messages in other clients immediately. If you move out of the private channel to the global view of all channels and then back, all messages are marked black (sent).

**Comment by rafaelks** [comment]
@dhoeld Got it! Is this always reproducible? Did you do something special to reproduce this?
